ci-operator/step-registry/assisted/baremetal/gather/assisted-baremetal-gather-commands.sh

```diff
@@ -43,7 +43,7 @@ source /root/config
 
 # Get sosreport including sar data
 sos report --batch --tmp-dir /tmp/artifacts --all-logs \
-  -o memory,container_log,filesys,kvm,libvirt,logs,networkmanager,networking,podman,processor,rpm,sar,virsh,yum \
+  -o memory,container_log,filesys,kvm,libvirt,logs,networkmanager,networking,podman,processor,rpm,sar,virsh,yum,cloud_init \
```

Comment:
we need it added to the ``assisted-baremetal-operator-gather`` step, no?

Reply:
or maybe even just to the ``assisted-baremetal-operator-gather`` step, unless we want to add it to all gather steps (``baremetalds-sno-gather`` included)

Reply:
:man_facepalming:

Reply:
I moved it to `assisted-baremetal-operator-gather`!